openslides/agenda/projector.py

```diff
@@ -73,4 +78,4 @@ def get_requirements(self, config_entry):
                     yield ProjectorRequirement(
                         view_class=speaker.user.get_view_class(),
                         view_action='retrieve',
-                        pk=str(speaker.user_id))
```

Comment:
I thought that `speaker.user_id` returns the id of the user without hitting the database about the user itself, so not for `speaker.user.pk`. @ostcar Am I right?